doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -7404,11 +7404,11 @@ \subsubsection{Inputs}\label{inputs-31}
 
 \paragraph{Field: Speed \textless{}x\textgreater{} Reference Unit Rated Air Flow Rate}\label{field-speed-x-reference-unit-rated-air-flow-rate-2}
 
-This numeric field defines the volumetric air flow rate, in m\(^{3}\)per second, across the cooling coil at rated conditions for Speed \textless{}x\textgreater{} operation. The value entered here should be directly from the Reference Unit data, corresponding to the given gross rated total cooling capacity and gross rated cooling COP at the speed, as above.
+This numeric field defines the volumetric air flow rate, in m\(^{3}\) per second, across the cooling coil at rated conditions for Speed \textless{}x\textgreater{} operation. The value entered here should be directly from the Reference Unit data, corresponding to the given gross rated total cooling capacity and gross rated cooling COP at the speed, as above.
```

Comment:
Ideally, these should be updated to use the appropriate `\si` macros, but I won't hold this up for that.

Reply:
Got you, I can make these changes.

Also previous part I thought the error was from `\volumeFlowRate`. Will check out the `\volumeFlowRateperWatt` @mjwitte